db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,40 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_authentications do |t|
+      t.string :name
+      t.boolean :challenge
```

Comment:
also, not knowing the semantics, but do the booleans say which kind of auth this is? If this is the case I would rather go for an Enum like `authentication_type`